voxel51/apps/api.py

```diff
@@ -35,6 +35,16 @@ class ApplicationAPI(API):
     platform users. In order to use these user-specific methods, however, you
     must first activate a user via :func:`ApplicationAPI.with_user`.
 
+    Using this API requires a valid application token. The following strategy
+    is used to locate your active application token:
```

Comment:
Might make more sense to say "Any of the following strategies can be used to locate your active application token"

Reply:
This list is showing the order precedence used to locate the active token. I updated the text to clarify